layouts/community/list.html

```diff
@@ -29,4 +29,4 @@ <h1>{{ .Title }}</h1>
 	{{ end }}
 	{{ partial "page-meta-lastmod.html" . }}
 </div>
-{{ end }}
+{{ end }}
```

Comment:
do not modify this file if there is no change.

Reply:
I have deleted the commit, thanks for the suggestion！